HW1/HW1Q4.py

```diff
@@ -0,0 +1,7 @@
+n = int(input('n = '))
+m = int(input('m = '))
+k = int(input('k = '))
+if k < m*n and k % m == 0 or k < m*n and k % n == 0:
```

Comment:
Забыли скобки и зачем повторили условие?
```suggestion
if k < m * n and (k % m == 0 or k % n == 0):
```
Также можно расширить первое условие по нижней границе допустимых значений переменной `k`